docs/modeling/generated-properties.rst

```diff
@@ -0,0 +1,99 @@
+.. include:: /rc1-notice.txt
+
+Generated Properties (Identity, Computed, Store Defaults, etc.)
+===============================================================
+
+.. contents:: In this article:
+    :depth: 3
+
+Value Generation Patterns
+-------------------------
+There are three value generation patterns that can be used for properties.
+
+No value generation
+^^^^^^^^^^^^^^^^^^^
+No value generation means that you will always supply a valid value to be saved to the database. This valid value must be assigned to new entities before they are added to the context.
+* **Value generated on add** means that if you don't specify a value, one will be generated for you.
```

Comment:
Duplicated from [this line](https://github.com/aspnet/EntityFramework.Docs/pull/63/files#diff-73bff1fb2b23f025c5afceb22f8f62eeR21)?